notebooks/11-xarray-intro.md

```diff
@@ -790,3 +684,69 @@ ll = ax.imshow(ndvi.values, cmap="RdYlGn", norm=div_norm)
 fig.colorbar(ll);
```

Comment:
TODO for if there is time: it might be good to include some more instructions + hints to be able to create those figures

Reply:
I adjusted the exercise with some more background and detailed steps on how to apply the normalization.